Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -84,13 +84,14 @@ usa,[*],option,,USD,100,0.01,1
 
 usa,[*],equity,,USD,1,0.01,1
 
+usa,BCF,future,,USD,50,0.25,1
 usa,ZC,future,,USD,50,0.25,1
 usa,ZW,future,,USD,50,0.25,1
 usa,ZS,future,,USD,50,0.25,1
 usa,ZM,future,,USD,100,0.1,1
 usa,ZL,future,,USD,600,0.01,1
 usa,ZO,future,,USD,50,0.25,1
-usa,ZT,future,,USD,2000,0.0078125,1
+usa,ZT,future,,USD,2000,0.00390625,1
```

Comment:
Can you please describe how you got this value?

Reply:
From the sample data you've provided, I've verified that this new entry in symbol-properties-database is the correct value.

I did some digging into bonds and its terminology, and found out that "Face value at maturity of $200,000" means the par value of $200,000 in bonds, where par value is usually $100. With that being said, the par value of the bond is $2000 USD ($200000 / $100), which is our contract multiplier.

Following the instructions on the CME page, I verified that the tick value is in fact 1/256 because the sample data you've provided has minimum increments of 1/256ths. Not only that, if we multiply the Contract Multiplier and Minimum Price Variation, we should have the USD value per contract ($2000 * (1/256) = $7.8125). In addition, if the price of the future goes up by $1, the underlying value would have gone up by $2000 (per contract). Because the minimum price increment is 1/256 (i.e. 0.00390625), minimum_price_variation should be 0.00390625